src/containerapp/azext_containerapp/_clients.py

```diff
@@ -690,6 +690,22 @@ def check_name_availability(cls, cmd, resource_group_name, name, name_availabili
         return r.json()
 
 
+    @classmethod
+    def get_auth_token(cls, cmd, resource_group_name, name):
```

Comment:
Confused why is this repeated? same as line 405.

Reply:
One is for environment tokens and one is for container app tokens -- note that one is under the containerapp environment class and the other is under the containerapp class

Reply:
The API URL is different for each as well